easybuild/easyconfigs/c/casacore/casacore-3.5.0-foss-2023b.eb

```diff
@@ -11,12 +11,23 @@ Thus user's can update them.
 
 toolchain = {'name': 'foss', 'version': '2023b'}
 
-source_urls = ['https://github.com/%(name)s/%(name)s/archive']
-sources = ['v%(version)s.tar.gz']
+sources = [
+    {
+        'filename': 'v%(version)s.tar.gz',
+        'source_urls': ['https://github.com/%(name)s/%(name)s/archive'],
+    },
+    {
+        'filename': 'WSRT_Measures.ztar',
+        'source_urls': ['ftp://anonymous@ftp.astron.nl/outgoing/Measures'],
+        'extract_cmd': 'tar xfvz %s --one-top-level=%(installdir)s/data',
```

Comment:
I don't think this actually works correctly, since an existing installation directory will be removed at the start of the install step.

We should instead extract into the build directory, and then copy over `data` from `%(builddir)s` to `%(installdir)s` via `postinstallcmds`.

This also highlights that we're missing something in `sanity_check_paths`.

Reply:
Interesting. I wasn't sure either, I was actually wondering if `%(installdir)s` was already set at this stage. But I simply tried, and it _did_ work for me. I did already have the software installed in did a `--rebuild`, but that shouldn't matter. In fact, I think later I tried to remove the dirs first and then re-install - I believe that also worked.

Anyway, I did consider your option too. Even though this works, maybe it's better to change it anyway: it feels a bit hacky to extract directly into the installdir in the extract step indeed...

Reply:
Edit: maybe I checked the wrong path, and it didn't work. I can't seem to find that data dir now. Anyway, I'll change it.

Reply:
Done, see https://github.com/easybuilders/easybuild-easyconfigs/pull/22201/commits/601e53761c987897189606cd2aa12a16277fa679